### PR TITLE
Add Capstone Logistics

### PR DESCRIPTION
**Pull request by jmaddox19**
This is still a WIP. I'll bring up some things we need to decide during standup tomorrow

### CODE REVIEW

**Comment by beccaelenzil** [comment]
This info looks good! Is this doc meant to be the student facing? If so, I might make a few changes to the layout and make the voice consistent.

File: capstone-overview/logistics.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Capstone Logistics to Review
```
Comment (beccaelenzil) [inline capstone-overview/logistics.md:1]: This info looks good! Is this doc meant to be the student facing? If so, I might make a few changes to the layout and make the voice consistent.

File: capstone-overview/logistics.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+1. Days off: The only holiday during capstone period is President's Day on February 15. Instructors will not be available to support on that day.
+1. Core hours: You can expect a staff member or volunteer TA to be holding office hours 9:30-12:00 and 1:00-3:00 at a minimum.
```
Comment (beccaelenzil) [inline capstone-overview/logistics.md:4]: Back when we were in person, students generally worked from home Tues/Thurs. I am wondering if this level of support is feasible for us as a time and necessary.

File: capstone-overview/logistics.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+## Capstone Logistics to Review
+
+1. **Days off:** The only holiday during capstone period is President's Day on February 15. Instructors will not be available to support on that day.
+1. **Standup Groups:** PMs and Standup groups will be assigned on Tuesday January 26. Standups should occur daily starting Wednesday January 27. Each group is able to choose its own time to meet together.
+1. **Staff & TA Support:** In addition to your Standup support groups and your assigned PM, Instruction Staff and TAs are happy to support with your projects.  Checkout the [Capstone Support Form spreadsheet](https://docs.google.com/spreadsheets/d/1QKlTPp5TX7YpZC3hO091VSd_M_eklNHagkJQ164bIic/edit?usp=sharing) for information of who could be most helpful with the technologies you're using.
```
Comment (beccaelenzil) [inline capstone-overview/logistics.md:5]:
```suggestion
1. **Staff & TA Support:** In addition to your Standup support groups and your assigned PM, Instruction Staff and TAs are happy to support with your projects.  Checkout the [Capstone Support Form spreadsheet](https://docs.google.com/spreadsheets/d/1QKlTPp5TX7YpZC3hO091VSd_M_eklNHagkJQ164bIic/edit?usp=sharing) for information of who could be most helpful with the technologies you're using.
* You can refer to the TA calendar for general TA and instructor availability.
```

Comment (beccaelenzil) [reply capstone-overview/logistics.md:5]: Not sure, what do you think?

File: capstone-overview/logistics.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+1. **Standup Form:** There will be a stand-up form you must fill out every day before your group's standup. Your PM will be checking that it was submitted. This helps you and your PM track your progress and helps ensure you're prepared to give a succinct, clear standup update to your group.
```
Comment (beccaelenzil) [inline capstone-overview/logistics.md:6]: Do you know where to find the link for this?

File: capstone-overview/logistics.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+1. **Week One:** The first week (Jan 25 - 29) is intended to be primarily learning focused. We encourage people to spend much of that time learning and experimenting with the languages and frameworks they plan to use. Not much progress on building features for the project is necessarily expected during this first week.
+1. **Recurring things:**
+    - PMs will meet with each individual weekly
```
Comment (beccaelenzil) [inline capstone-overview/logistics.md:9]: I'm trying to figure out the words to say that 1:1s can optionally start Thursday 1/28, but I'm having trouble wording it

Comment (jmaddox19) [reply capstone-overview/logistics.md:10]: Yeah I ran into the same problem. Haha. 
I decided that's just something we can explain to them in a timely manner, like on Tuesday or Wed when it feels more relevant

Comment (jmaddox19) [reply capstone-overview/logistics.md:10]: Hope that works! Thanks for looking again

**Pull request merged**